app/assets/stylesheets/application/contributors.scss

```diff
@@ -1,9 +1,34 @@
-.hideContent {
-  display: none;
-}
+.contributorModal-body {
```

Comment:
We haven't had a consensus on using BEM, though I think it's awesome. For now, can you change these class names to camel case? Happy to discuss more outside of this PR :)